### DISCUSSION TITLE
两点建议

### DISCUSSION THREAD

**Discussion by AnnAngela** [takayama-lily/oicq#287] [Ideas]
1. 能否在GroupRequestEvent.approve里传入拒绝理由？
2. 能否在GroupRequestEvent里单列一个属性用以**仅**传入申请问题的答案？（现在的comment会把问题和答案都传进来，不是很方便）
![image](https://user-images.githubusercontent.com/9762652/148629879-5801d81d-de5a-4f65-84b8-c0161c1985df.png)

**Comment by takayama-lily** [comment]
1. 大概可以
2. 不可以，原始数据就粘在一起

**Reply by AnnAngela** [reply]
感谢解答，那麻烦大佬安排一下第一点，谢谢！
